common/testing/testhooks/constants.go

```diff
@@ -0,0 +1,31 @@
+// The MIT License
+//
+// Copyright (c) 2024 Temporal Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package testhooks
+
+const (
+	MatchingDisableSyncMatch      = "matching.disableSyncMatch"
+	MatchingLBForceReadPartition  = "matching.lbForceReadPartition"
+	MatchingLBForceWritePartition = "matching.lbForceWritePartition"
+
+	UpdateWithStartInBetweenLockAndStart = "history.updateWithStartInBetweenLockAndStart"
```

Comment:
Could we use `iota` instead? Since the identifiers don't need to be stable; it would save us some typing and effort to keep things consistent.

Counter point could be that we lose the ability to print them. Which we don't do right now; but might in the future.

Reply:
Print them as `int` should be ok too. Anyway, adding `Stringer` implementation later is not a big deal.